### DISCUSSION TITLE
Is there a way to add file comment on PR review?

### DISCUSSION THREAD

**Discussion by Eryx5502** [pwntester/octo.nvim#562] [Q&A]
Is there a way to add a "file comment" in a PR review? I do know how to add comments to specific lines, but not sure how to add one applying to the whole file (that is, as in github webpage, see screenshot below)?

![imagen](https://github.com/pwntester/octo.nvim/assets/32964371/0fda8893-52e7-47e5-a59b-9afffc7f15c6)

**Comment by ysulyma** [comment]
Came here to ask the same question.

**Reply by williambdean** [reply]
Would you be interested in making a PR @ysulyma ?

**Comment by williambdean** [comment]
At the moment, I don't think that there is. But it would be a great addition!

I suspect that this mutation can be used: https://docs.github.com/en/graphql/reference/mutations#addpullrequestreviewthread

It seems that the one currently being used in deprecating many inputs including `path`.
That mutation is already used in two spots. For instance: https://github.com/pwntester/octo.nvim/blob/5579f207e50cce726a2a5a460f48093ee7558fdb/lua/octo/gh/graphql.lua?plain=1#L324-L345

Would someone be open to making a PR?

**Reply by BlakeLReeves** [reply]
I may look into working on this and opening a pr. I would use this daily.
